src/components/TodoHeader/TodoHeader.tsx

```diff
@@ -0,0 +1,70 @@
+import { useRef, useEffect } from 'react';
+import { Errors } from '../../types/Errors';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todos: Todo[],
+  onAddTodo: (title: string) => void,
+  setError: (error: Errors) => void,
+  isLoading: boolean,
+  todoTitle: string,
+  setTodoTitle: (title: string) => void,
+};
+
+export const TodoHeader: React.FC<Props> = ({
+  onAddTodo,
+  setError,
+  isLoading,
+  todos,
+  todoTitle,
+  setTodoTitle,
+}) => {
+  const inputRef = useRef<HTMLInputElement>(null);
+
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event?.preventDefault();
+    if (!todoTitle
+      || todoTitle.split(' ').filter(char => char.length !== 0).length === 0) {
```

Comment:
you can use [`trim()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/trim)

Reply:
Not fixed 😢